app/views/components/gds/linkButton.scala.html

```diff
@@ -0,0 +1,23 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(messageKey: String, call: Call)(implicit messages: Messages)
```

Comment:
if we pass text here directly we don't have to pass messages here

Reply:
True.  But all the other "buttons" we've created to date in this package pass the key